### PR TITLE
Feature sonos controls and youtube

### PR DESCRIPTION
**Pull request by jballant**
Adding youtube playing functionality and sonos volume controls.

### CODE REVIEW

**Comment by jballant** [comment]
@andromedado, would this be a Major version change? I removed the spotify volume controls (and now added sonos volume controls instead).

**Comment by cmckendry** [comment]
Can we actually start with maxVol set lower and then bump it up once we're more confident? 

To keep Spotify and Youtube in the same realm I suspect we'll need to set Spotify pretty high, and Spotify at 95 + Sonos at 50 is probably a bit much.

Maybe 35 - 40 just for experimenting?

**Comment by jballant** [comment]
You can set an env var: HUBOT_SONOS_MAX_VOLUME and then use whatever you want. You can start with 15 if you like. If you want I can edit the default to be lower too.

File: src/scripts/spot.js [hunk 1]
```diff
@@ -604,6 +605,7 @@ module.exports = function (robot) {
         }
         return setVolume(adi, message);
     });
+*/
```
Comment (andromedado) [inline src/scripts/spot.js:608]: comments... :disappointed:

**Pull request merged**